targets/TARGET_Maxim/TARGET_MAX32620/TARGET_MAX32620HSP/PinNames.h

```diff
@@ -126,6 +126,10 @@ typedef enum {
     SW1 = P2_1,
     SW2 = NOT_CONNECTED,
     SW3 = NOT_CONNECTED,
+    // Standardized button names
+    BUTTON1 = SW1,
+    BUTTON2 = SW2,
+    BUTTON3 = SW3,
```

Comment:
This is an interesting one actually,  since `SW2` and `SW3` aren't actually connected. In this case I'd say let's not add any `BUTTONx` definitions. I'd rather this fail at compile time. What do you think @0xc0170?